app/templates/briefs/_brief_response_data.html

```diff
@@ -27,17 +27,19 @@
             {% elif item.type == "dynamic_list" %}
               {% for question in item.questions %}
                   {% if question.type == "boolean" %}
-                    {% if question._data['followup'] %}
+                    {% if question._data['values_followup'] %}
```

Comment:
`question.values_followup` is a `@property`, so it doesn't exist in `_data`, I think we can still use `followup` for this check, although I'm not sure why we're getting it from `._data` as opposed to the question itself, @idavidmcdonald?